components/service/glean/src/main/java/mozilla/components/service/glean/testing/GleanTestRule.kt

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.service.glean.testing
+
+import android.content.Context
+import androidx.work.testing.WorkManagerTestInitHelper
+import mozilla.components.service.glean.Glean
+import mozilla.components.service.glean.config.Configuration
+import org.junit.rules.TestWatcher
+import org.junit.runner.Description
+
+/**
+ * This is implements a JUnit rule for writing tests for Glean SDK metrics.
```

Comment:
Nit: "This implements"

Reply:
Thanks! [The docs](https://mozilla.github.io/glean/book/user/testing-metrics.html) are built off a separate repo, we're changing them as part of mozilla/glean#211 . I will make sure that lands before this one :)